apps/auth_zero/auth0backend.py

```diff
@@ -1,5 +1,5 @@
-from urllib import request
-
+import requests
+from django.conf import settings
```

Comment:
Codacy found an issue: [third party import "from django.conf import settings" should be placed before "from jose import jwt"](https://app.codacy.com/gh/Xcov19/covidX/pullRequest?prid=6329042)